README.md

```diff
@@ -63,14 +62,19 @@ For more, see [responsibilities](#responsibilities).
 + This must be a separate company from Founders & Coders CIC,
 so that F&C can focus solely on the ***mission***;
 Providing ***great technical learning***, ***free*** and ***open source***.
-+ The company will and should sit in on F&C meetings but *not*
-have the power to dictate what happens in the school.
++ The company will and should participate in F&C meetings but _not_
+have the power to _dictate_ what happens in the school.
```

Comment:
The only times I've seen this _not_ work is when the underscores are in the _middle_ of a word, so I hadn't realised this.

I just find it **significantly more time consuming** to understand what's going on in the markdown with `***` than I do with `**_`, hence my preference. The sentence below as an example.

```markdown
An ***alternative solution*** is therefore ***required*** to these problems
which will allow **F&C students/alumni** to ***continue learning*** (*by doing*)
after the 16 week course while providing a ***sustainable income***
to fund the mission of **F&C**.
```